modules/govuk_solr6/manifests/install.pp

```diff
@@ -0,0 +1,54 @@
+# == Class: govuk_solr::install
+#
+# Installs the solr package
+#
+class govuk_solr6::install (
+  $version  = '6.4.2',
+  $base_url = 'https://archive.apache.org/dist/lucene/solr/',
```

Comment:
I'm not an expert but maybe you need to have a copy of solr on our own apt server: https://docs.publishing.service.gov.uk/manual/debian-packaging.html

Reply:
If only we could use a debian package... the debian and ubuntu packages of solr are 3.x, so this script has to do all the installation manually. It's essentially a highly modified copy of the existing `govuk_solr` module, copied because i think we're going to need to keep the old solr working on e.g. production for quite a while until we're ready to migrate...

Reply:
So reading further, maybe I _should_ create a debian package from this and add it to https://github.com/alphagov/packager/tree/master/fpm/recipes